cmd/meroxa/root/connectors/connectors.go

```diff
@@ -24,7 +24,8 @@ import (
 type Connectors struct{}
 
 var (
-	_ builder.CommandWithAliases = (*Connectors)(nil)
+	_ builder.CommandWithAliases     = (*Connectors)(nil)
+	_ builder.CommandWithSubCommands = (*Connectors)(nil)
```

Comment:
Nice, while we're at it we're also missing `CommandWithDocs`.
```suggestion
	_ builder.CommandWithSubCommands = (*Connectors)(nil)
	_ builder.CommandWithDocs        = (*Connectors)(nil)
```